pr-data.csv

```diff
@@ -2790,7 +2790,7 @@ https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1a
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.AccumulateTest.testOneAccumulateOnPattern,ID,DeveloperWontFix,,https://github.com/apache/incubator-kie-drools/pull/6119
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.AccumulateTest.testSemicolonMissingInInit,ID,DeveloperWontFix,,https://github.com/apache/incubator-kie-drools/pull/6119
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.AccumulateTest.testSubnetworkTuple,ID,,,
-https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.AccumulateTest.testTwoAccumulatesOnPattern,ID,Opened,https://github.com/apache/incubator-kie-drools/pull/6465,
+https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.AccumulateTest.testTwoAccumulatesOnPattern,ID,,,Rejected due to developer already working with PR which came after: https://github.com/apache/incubator-kie-drools/pull/6465
```

Comment:
Strictly speaking, this should have changed status from `Opened` to `Rejected`, but it may not matter if it gets hopefully soon `Accepted` for the other PR.